### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi @rockambole86, @FromDoppler/sap, and team,

This PR has two objectives:
* Adding a little more information to our README.md
* Allow me to test if CI is working properly

Could you take a look?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,55 @@
...
+            "path?": "/accounts/doppler/22235/invoices?from=2020-02-01T00:00:00Z&to=2020-07-01T00:00:00Z"
+        }
+    ]
+}
+​
+###
+GET /accounts/doppler/22235/invoices/501_20200508_130218.pdf
+Authorization: Bearer {{token-jwt}}
+​​
+// retorna el PDF con content type de pdf
```
Comment (nreal) [inline README.md:49]: Maybe this text should be in English but I guess that it is temporal

Comment (andresmoschini) [reply README.md:49]:
```suggestion
// Return PDF content with `application/pdf` content type
```

**Comment by nreal** [comment]
👍

**Comment by rockambole86** [comment]
Silly question... shouldn't the samples use `billing` word somewhere in the URL path?
At first impression, we would be using the doppler `accounts` controller. Am I confused?

**Comment by andresmoschini** [comment]
> Silly question... shouldn't the samples use `billing` word somewhere in the URL path?
> At first impression, we would be using the doppler `accounts` controller. Am I confused?

I think that `billing` is the general name because this API will be grown with time. By the moment `billing` will be at the beginning: `GET https://apis.fromdoppler.com/billing/accounts/relay/2124545/invoices?from=2020-01-01T00:00:00Z&to=2020-07-01T00:00:00Z`

Does it make sense?

**Pull request merged**